src/allmydata/web/status.py

```diff
@@ -868,73 +868,113 @@ def render_server_timings(self, ctx, data):
             l[T.li["[%s]: %s" % (server.get_name(), times_s)]]
         return T.li["Per-Server Response Times: ", l]
 
-class MapupdateStatusPage(rend.Page, RateAndTimeMixin):
-    docFactory = getxmlfile("map-update-status.xhtml")
 
-    def __init__(self, data):
-        rend.Page.__init__(self, data)
-        self.update_status = data
+# Renders "/status/mapupdate-%d"
```

Comment:
probably better as a docstring instead of bare comment?

Reply:
Made this change in c184888.